packages/authentication-oauth/src/index.ts

```diff
@@ -43,16 +43,16 @@ export const setup = (options: OauthSetupSettings) => (app: Application) => {
 
   const grant = merge({
     defaults: {
-      path,
-      host,
-      protocol,
-      transport: 'session'
+      prefix,
+      origin: `${protocol}://${host}`,
+      transport: 'session',
+      response: ['tokens', 'raw', 'profile']
```

Comment:
`['tokens', 'raw']` should be the default - not sure whether `tokens` would be enough?

`profile` is added, to make grant make the request to `profile_url`.